File: frontend/src/modules/cruise-schedule/models/CruiseFormDto.ts
export type CruiseManagersTeamDto = {
  mainCruiseManagerId: string;
  mainDeputyManagerId: string;
};

export type CruiseFormDto = {
  startDate: string;
  endDate: string;
  managersTeam: CruiseManagersTeamDto;
  cruiseApplicationsIds: string[];
};

File: backend/ResearchCruiseApp/Application/Models/DTOs/Cruises/CruiseFormDto.cs
namespace ResearchCruiseApp.Application.Models.DTOs.Cruises;

public class CruiseFormDto
{
    public string StartDate { get; set; } = null!;

    public string EndDate { get; set; } = null!;

    public CruiseManagersTeamDto ManagersTeam { get; set; }

    public List<Guid> CruiseApplicationsIds { get; set; } = [];

    public string? Title { get; set; }

    public bool ShipUnavailable { get; set; } = false;
}
